ietf/templates/release/about.html

```diff
@@ -1,5 +1,6 @@
 {# Copyright The IETF Trust 2015-2022, All Rights Reserved #}
 {% extends "base.html" %}
+{# Copyright The IETF Trust 2015-2022, All Rights Reserved #}
```

Comment:
Another add of a line that already exists.

Reply:
removing